src/sw.ts

```diff
@@ -460,7 +465,7 @@ async function fetchHandler ({ path, request, event }: FetchHandlerArg): Promise
     const response = await verifiedFetch(verifiedFetchUrl, {
       signal,
       headers,
-      // TODO redirect: 'manual', // enable when http urls are supported by verified-fetch: https://github.com/ipfs-shipyard/helia-service-worker-gateway/issues/62#issuecomment-1977661456
+      redirect: 'manual',
```

Comment:
finally done with this todo

Reply:
That would close https://github.com/ipfs-shipyard/helia-service-worker-gateway/issues/62

Reply:
i'll add an e2e test prior to closing #62

Reply:
one last thing blocking fix of #62: https://github.com/ipfs/helia-verified-fetch/pull/33

it was previously only working for subpath directories.